tests/test_shardformer/test_model/_utils.py

```diff
@@ -246,3 +248,17 @@ def check_grad(org_model: Module,
         assert torch.allclose(
             org_grad.float(), shard_grad.float(), rtol=rtol, atol=atol
         ), f"error attribute '{suffix}', orgin model grad is not equal to shard model grad\n{org_grad}\n{shard_grad}"
+
+
+def unwrap_model(module: Module,
+                 base_model_class_name: Optional[str] = None,
+                 base_model_attribute_name: Optional[str] = None):
+    if isinstance(module, HybridParallelModule):
```

Comment:
Model wrapper class already has `unwrap()` method. I think you should overwrite this method in `HybridParallelModule`.

Reply:
This util function is only for the convenience of testing, so I'm putting it under testing folder.

Reply:
This util function is only for testing usage, but it makes sense to overwrite the unwrap() method in HybridParallelModule to deal with DDP cases.